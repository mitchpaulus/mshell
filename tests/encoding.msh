"°" wl
$"°" wl # Format strings are lexed/parsed with some independent code so also need this test.
