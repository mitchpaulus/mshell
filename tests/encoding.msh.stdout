c2 b0
Â°
